OMCompiler/SimulationRuntime/c/util/integer_array.c

```diff
@@ -93,17 +94,7 @@ void alloc_integer_array_data(integer_array_t* a)
 
 void copy_integer_array_data(const integer_array_t source, integer_array_t* dest)
 {
```

Comment:
Why these wrapper functions? copy_integer_array_data is just calling integer_array_copy_data...